FlowCal/io.py

```diff
@@ -972,6 +976,17 @@ def analysis(self):
         return self._analysis
 
     @property
+    def data_type(self):
+        """
+        Type of data in the FCS file's DATA segment.
+
+        `data_type` is 'I' if the data type is integer, 'F' for floating
+        point, and 'D' for double.
+
+        """
+        return self._data_type
```

Comment:
this `data_type` attribute is just for convenience? easier than making the user access the TEXT segment dictionary? was there a particular motivation for exposing this but not other things? (i don't really have strong feelings either way, just curious)